reference/test/reorder/scaled_reordered.cpp

```diff
@@ -364,6 +364,8 @@ TYPED_TEST(ScaledReordered, AppliesWithRcmReordering)
 TYPED_TEST(ScaledReordered, SolvesSingleRhsWithOnlyInnerOperator)
 {
     using SR = typename TestFixture::SR;
+    using value_type = typename TestFixture::value_type;
+    SKIP_IF_HALF(value_type);
```

Comment:
The system is fairly small, so I wouid expect that the system can be solved with half precision. Is that not the case?

Reply:
We need the scaling to make it work. I have deleted skip_if_half in the following tests